test/models/ETMPPayloadSpec.scala

```diff
@@ -288,6 +371,90 @@ class ETMPPayloadSpec extends AnyWordSpec with Matchers {
     )
   )
 
+  val etmpPayloadModelWithVATOverview: ETMPPayload = ETMPPayload(
+    pointsTotal = 1,
+    lateSubmissions = 1,
+    adjustmentPointsTotal = 1,
+    fixedPenaltyAmount = 200.00,
+    penaltyAmountsTotal = 400.00,
+    otherPenalties = Some(false),
+    vatOverview = Some(
+      Seq(
+        OverviewElement(
+          `type` = AmountTypeEnum.VAT,
+          amount = 100.00,
+          estimatedInterest = Some(10.00),
+          crystalizedInterest = Some(10.00)
+        ),
+        OverviewElement(
+          `type` = AmountTypeEnum.Central_Assessment,
+          amount = 100.00,
+          estimatedInterest = Some(10.00),
+          crystalizedInterest = Some(10.00)
+        )
+      )
+    ),
+    penaltyPointsThreshold = 4,
+    penaltyPoints = Seq(
+      PenaltyPoint(
+        `type` = PenaltyTypeEnum.Financial,
+        number = "2",
+        id = "123456790",
+        appealStatus = Some(AppealStatusEnum.Under_Review),
+        dateCreated = sampleDate,
+        dateExpired = Some(sampleDate),
+        status = PointStatusEnum.Active,
+        reason = None,
+        period = Some(PenaltyPeriod(
+          startDate = sampleDate,
+          endDate = sampleDate,
+          submission = Submission(
+            dueDate = sampleDate,
+            submittedDate = Some(sampleDate),
+            status = SubmissionStatusEnum.Submitted
+          )
+        )),
+        communications = Seq(
+          Communication(
+            `type` = CommunicationTypeEnum.secureMessage,
+            dateSent = sampleDate,
+            documentId = "1234567890"
+          )
+        ),
+        financial = Some(Financial(
+          amountDue = 400.00,
+          dueDate = sampleDate
+        ))
+      ),
+      PenaltyPoint(
+        `type` = PenaltyTypeEnum.Point,
+        number = "1",
+        id = "123456789",
+        appealStatus = None,
+        dateCreated = sampleDate,
+        dateExpired = Some(sampleDate),
+        status = PointStatusEnum.Active,
+        reason = Some("reason"),
+        period = Some(PenaltyPeriod(
+          startDate = sampleDate,
+          endDate = sampleDate,
+          submission = Submission(
+            dueDate = sampleDate,
+            submittedDate = Some(sampleDate),
+            status = SubmissionStatusEnum.Submitted
+          )
+        )),
+        communications = Seq(
+          Communication(
+            `type` = CommunicationTypeEnum.letter,
+            dateSent = sampleDate,
+            documentId = "1234567890")
+        ),
+        financial = None
+      )
+    )
+  )
+
```

Comment:
Could you add some tests in here? I think my backend PR has some

Reply:
Added tests